src/modules/extendedCallWindow/i18n/nb_NO.js

```diff
@@ -0,0 +1,169 @@
+            arff: 'ARFF',
+            hems: 'HEMS',
+            kdow_orgl: 'EMS Chief',
+            grtw: 'Mass Casualty Unit',
+            fustw: 'Patrol Car',
+            police_motorcycle: 'Police Motorcycle',
+            fustw_or_police_motorcycle: 'Patrol car or Police Motorcycle',
+            polizeihubschrauber: 'Police helicopter',
+            swat: 'SWAT',
+            swat_armored_vehicle: 'SWAT Armoured Vehicle',
+            swat_suv: 'SWAT SUV',
+            k9: 'k-9 Unit',
+            gkw: 'Utility Truck',
+            boot: 'Boats (General)',
+            brush_vehicle: 'Wildland fire vehicle',
+            brush_truck: 'Wildland fire engine',
+            brush_truck_1: 'Type 3 engine',
+            brush_truck_2: 'Type 5 engine',
+            brush_truck_3: 'Type 7 engine',
+            brush_truck_4: 'Type 4 engine',
+            brush_truck_5: 'Type 6 engine',
+            dozer_trailer: 'Dozer Trailer',
+            fire_aviation: 'Fire aviation',
+            water_drop_helicopter: 'Water drop helicopter',
+            air_tanker: 'Air tanker',
+            heavy_aur_tanker: 'Heavy air tanker',
+            crew_carrier: 'Crew Carrier',
+            fbi_unit: 'FBI Unit',
+            fbi_investigation_unit: 'FBI Investigation Unit',
+            fbi_mobile_command: 'FBI Mobile Command',
+            fbi_bomb_tech: 'FBI Bomb Tech',
+            fbi_drone: 'FBI Surveillance Drone',
+            sheriff_unit: 'Sheriff Unit',
+        },
+    },
+    generationDate: {
+        inputFormat: 'MMMM DD., HH:mm',
+    },
+    enhancedMissingVehicles: {
+        vehicle: 'Vehicle type',
+        missing: 'Missing on mission',
+        driving: 'Approach',
+        total: 'Still needed',
+        selected: 'Selected',
+        vehiclesByRequirement: {
+            [/^Fire truck(s)$/]: [0, 1, 13, 18, 30, 33],
+            [/^Platform truck(s)$/]: [2, 13],
+            [/^Battalion chief unit(s)$/]: [3, 12],
+            [/^Heavy Rescue Vehicle(s)$/]: [4, 8, 18, 21],
+            [/^Mobile air$/]: [6],
+            [/^Water Tanker$/]: [7, 33],
+            [/^mcv$/]: [12],
+            [/^Hazmat$/]: [9],
+            [/^Ambulance(s)$/]: [5, 27],
+            [/^Patrol Car(s)$/]: [10],
+            [/^HEMS$/]: [11],
+            [/^Police helicopter(s)$/]: [23],
+            [/^SWAT Personells$/]: [16, 26],
+            [/^ARFF Crahs Tender/]: [17],
+            [/^K-9 Unit(s)$/]: [19],
+            [/^Large Fireboat(s)$/]: [24],
+            [/^Large Rescue boat(s)$/]: [25],
+            [/^Wildland fire engine(s)$/]: [30, 31, 32, 38, 39, 40],
+            [/^Fire Aviation(s)$/]: [35, 36, 37],
+            [/^FBI Unit(s)$/]: [42],
+            [/^FBI Investigation Wagon(s)$/]: [43],
+            [/^FBI Mobile Command Center(s)$/]: [44],
+            [/^FBI Bomb Technician Vehicle(s)$/]: [45],
+            [/^FBI Surveillance Drone(s)$/]: [46],
+            [/^Sheriff Unit(s)$/]: [47],
+        },
+    },
```

Comment:
This are not the names that will be shown in the red bar.
So I think this wil result in an error for the table.

Reply:
no error, but table will not work. Can be done later actually. It is not easy to translate without having a basic translation on live system ;)